packages/services/hmi-server/src/main/resources/application.properties

```diff
@@ -45,8 +45,8 @@ skema/mp-rest/url=https://skema-py.staging.terarium.ai
 skema-rust/mp-rest/url=https://skema-rs.staging.terarium.ai
 skema-unified/mp-rest/url=https://skema-unified.staging.terarium.ai
 model-service/mp-rest/url=https://model-service.staging.terarium.ai
-%dev.data-service/mp-rest/url=http://${data-service-host:localhost}:${data-service-port:3020}
-%prod.data-service/mp-rest/url=http://${data-service}:3020
+%dev.data-service/mp-rest/url=http://${data-service-host:localhost}:${data-service-port:8001}
+%prod.data-service/mp-rest/url=http://${data-service}:8001
```

Comment:
I don't think this has been done, this is only on your local machine to do so.

Reply:
Yepp, this is a draft PR - just testing still was running into some errors had to run DB locally will revert when I open PR up